distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin-table-contents.adoc

```diff
@@ -0,0 +1,33 @@
+
+.[[org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin]]Catalog Metacard Ingest Network Plugin
```

Comment:
Kinda. This one is under `src/main/resources` to be included in the existing documentation. Everything under `src/main/jdocs` is what goes into the new template version, which is still in draft status